plugin_scores_cache.yml

```diff
@@ -145,12 +145,12 @@ fastlane-plugin-appicon:
     description: Generate required icon sizes and iconset from a master application
       icon
     details: 
-  :sha: 3255962f0ab5d6f38ccb98a361384172f6adf2e3ae97fb974dbebcc59038dfb1
-  :github_stars: 194
+  :sha: cec243fa2de67308651fa76786a0da3f57e8954c57e7a202374343da219f126c
+  :github_stars: 205
   :github_subscribers: 7
   :github_issues: 10
-  :github_forks: 15
-  :github_contributors: 14
+  :github_forks: 19
+  :github_contributors: 0
```

Comment:
We need to fix this someday :( I think it returns 0 when the API limit is reached or something, but didn't have time to take a closer look.